posts/packit_1_0_0_action_required/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Packit 1.0.0 countdown - action may be required"
+date: 2024-12-09T12:00:00+00:00
+authors: mmassari
+tags:
+  - configuration
+---
+
+After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!
+
+<!--truncate-->
+
+## What's New
+
+We will **remove all since long deprecated keys from the configuration file**.
+
+`packit-service` jobs are tight with `packit CLI` so be aware that for those projects still using deprecated keys in their `packit.yaml` project file we have created a migration pull request.
```

Comment:
I would maybe rephrase the beginning a bit, telling that the code regarding the configuration is placed in the `packit` package, therefore `packit-service` users will be affected as well.